hello-core-logic/displayBoardsMimicTrello.js

```diff
@@ -0,0 +1,53 @@
+const hello = {
+    'Tester Board': {
+      'To Do': ['Laundry', 'Buy Apples', 'Pay Phone Bill'],
+      'Doing': ['Laundry', 'Studying Javascript', 'Studying HTML', 'Studying Ruby'],
+      'Done': ['Laundry']
+    },
+    'Dreams': {
+      'Wish List': ['Conquer the Seven Kingdoms', 'Get my baby back', 'My hand needs to chill'],
+    },
+    'Cuisine': {
+      'Region':['African', 'European', 'Brazilian', 'Asian'] }
+};
+
+board = 'Tester Board';
+
+function displayBoard(boardName) {
+  let dash = "|---------------------";
+  let result = dash;
+  let count = 0;
+  for(let listName in boardName) {
+    if (boardName !== '') {
```

Comment:
As was explained for your `listBoard` function, there is no need for this if statement because it isn't actually checking anything